guides/hack/20-attributes/07-predefined-attributes.md

```diff
@@ -206,6 +208,17 @@ mock class can exist.
 
 Mock classes *cannot* extend types `vec`, `dict`, and `keyset`, or the Hack legacy types `Vector`, `Map`, and `Set`.
 
+## __Newable
+
+This attribute is used to annotate reified type parameters to ensure that they are only instantiated with classes on which `new` can be safely called.  A common pattern is:
+```Hack
+function f<<<__Newable>> reify T as A>(): T {
+  return new T();
+}
+```
```

Comment:
Can you:
- split this example out to a separate file like the others? This will alert us if/when this documentation becomes outdated
- add an example with parameters
- add an example where the attribute is not present (with error)
- add an example where the attribtue is present, but the class is not final or marked `__ConsistentConstruct`

Given the structure of the text, it might be best for the counter-examples to be in a separate file below the next paragraph

Reply:
Hi Fred, I have split the example out to a separate file, added a more realistic example, added a sentence about errors, and revised the overall text.  I believe that adding examples about errors would not add much to the text.  Let me know what you think!